.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,24 @@
+## Description
+A detailed description of your changes.
+
+
+## Motivation and Context
+Explanation of why these changes are being proposed, including any links to other relevant issues or pull requests.
+
+
+## How Has This Been Tested?
+A clear and concise description of your changes were tested.
+
+
+## Types of changes
```

Comment:
This feels like it would come before the testing section at least.  Or cover it in the `Description`

Reply:
I don't have a strong preference either way. This stuff is all optional anyway (or at least it's not enforced by the Github UI), though it's difficult to convey that clearly to contributors.

Feel free to suggest changes - I'll gladly to apply them if that's what everyone prefers. 😄

Reply:
Not a huge deal either way, as this can be updated over time.